Conch/lib/Conch/Controller/DeviceSettings.pm

```diff
@@ -36,8 +35,18 @@ overwritten
 =cut
 
 sub set_single ($c) {
-	my $body          = $c->req->json;
-	my $setting_key   = $c->param('key');
+	my $body = $c->req->json;
```

Comment:
I think there's a way to do this with less code by disabling extension based rendering. I'm investigating.